pygsti/algorithms/grasp.py

```diff
@@ -267,9 +293,18 @@ def _grasp_local_search(initial_solution, score_fn, elements, get_neighbors_fn,
     return currentSoln
 
 
-def run_grasp_iteration(elements, greedy_score_fn, rcl_fn, local_score_fn,
-                        get_neighbors_fn, feasible_threshold=None, feasible_fn=None,
-                        initial_elements=None, seed=None, verbosity=0):
+def run_grasp_iteration(
+    elements,
+    greedy_score_fn,
+    rcl_fn,
+    local_score_fn,
+    get_neighbors_fn,
+    feasible_threshold=None,
+    feasible_fn=None,
+    initial_elements=None,
+    rng=None,
```

Comment:
Here's another substantial change - adding `rng` argument again in place of the `seed` argument.